e2e/libs/node/utility.py

```diff
@@ -7,3 +9,31 @@ def get_node_by_name(node_name):
 def get_node_cpu_cores(node_name):
     node = get_node_by_name(node_name)
     return node.status.capacity['cpu']
+
+def list_node_names_by_role(role="worker"):
```

Comment:
There are similar functions in `utility/utility.py`. Is it possible to merge them?

Reply:
Sure

Reply:
https://github.com/longhorn/longhorn-tests/pull/1582